templates/access.html

```diff
@@ -136,16 +139,15 @@ <h4 id="exploit-description" style="margin-top:2px;text-align:center"></h4>
         }
     }
 
-    function showProperties(){
-        function propertiesCallback(data){
-            let cmd = b64DecodeUnicode(data[0].test);
-            let found = [], rxp = /{([^}]+)}/g, str = cmd, curMatch;
+    function setProperties() {
+        function propertiesCallback(ability) {
+            let found = [], rxp = /#{([^}]+)}/g, str = ability.executors[0].command, curMatch;
             while(curMatch = rxp.exec(str)) {
                 found.push(curMatch[1]);
             }
             $('#exploit-properties').empty();
-            $('#exploit-description').text(data[0].description);
-            $('#exploit-exploit').text(b64DecodeUnicode(data[0].test));
+            $('#exploit-description').text(ability.description);
+            $('#exploit-exploit').text(ability.executors[0].command);
```

Comment:
I momentarily forgot I was looking at javascript and thought this needed to be updated for the generator :D